deepmd/descriptor/se_a.py

```diff
@@ -797,12 +798,12 @@ def _filter(
                       bavg = bavg,
                       trainable = trainable,
                       suffix = "_"+str(type_i))
-                  if type_i == 0:
```

Comment:
Did we have a bug here? if `type_i == 0` and `(type_input, type_i)  in self.exclude_types` we had ret accumulated.